src/app/GitUI/CommandsDialogs/Menus/WorkingDirectoryToolStripSplitButton.cs

```diff
@@ -0,0 +1,181 @@
+using GitCommands;
+using GitCommands.UserRepositoryHistory;
+using GitExtensions.Extensibility.Git;
+using GitExtensions.Extensibility.Translations;
+using GitUI.CommandsDialogs.BrowseDialog;
+using ResourceManager;
+
+namespace GitUI.CommandsDialogs.Menus
+{
+    /// <summary>
+    ///  Represents a split button that contains the recent repositories.
+    /// </summary>
+    internal class WorkingDirectoryToolStripSplitButton : ToolStripSplitButton, ITranslate
+    {
+        private readonly TranslationString _noWorkingFolderText = new("No working directory");
+        private readonly TranslationString _configureWorkingDirMenu = new("&Configure this menu...");
```

Comment:
>BTW: Hotkeys are not displayed yet.

@mstv Just saw this comment.... fixed (one regression and one update).

Reply:
> ![image](https://private-user-images.githubusercontent.com/4403806/343034797-5aa6d9fb-b758-4062-a55b-b231aaa9e6a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0MjM0NjgsIm5iZiI6MTcxOTQyMzE2OCwicGF0aCI6Ii80NDAzODA2LzM0MzAzNDc5Ny01YWE2ZDlmYi1iNzU4LTQwNjItYTU1Yi1iMjMxYWFhOWU2YTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjZUMTczMjQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTczMjIzNzk3ZTBiMWQ2YjJhNTNjN2NmNTdiZWNjNjVlMTE0NzVlNTkxYmM5MDc0ZDM4MmY0ZjYyNTljOTI4YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ._S4Z3CJ6TaOtRVBtuJa4dQtcxmtPpZaFoAs85IZZQBc)

This is exactly why I requested the change: Please avoid duplicate mnemonics.

Reply:
The image didn't come through.

The mnemonics are exactly the same as in the other places, and this is exactly what we want because it is exactly the same functionality exposed from a different place.

![image](https://github.com/gitextensions/gitextensions/assets/4403806/4d59c8f3-0d25-46d2-a9f0-139a1c6d3bf0)
![image](https://github.com/gitextensions/gitextensions/assets/4403806/52731403-5a0d-4d23-b720-22e749fa7633)

![image](https://github.com/gitextensions/gitextensions/assets/4403806/c48ff546-9127-4b64-b7b6-f23da1a18d27)

Reply:
@RussKie it's seems that you are not talking about the same things. Hot keys are OK but @mstv was talking about mnemonics '&' that is on the 'C' in the new 'Close' menuitem like in the 'Configure this menu...'.That's why he asked you to change it for one.

Reply:
Yes, I completely missed that part, my apologies.

Reply:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/071ddb86-ae1b-43b2-ae19-8e4854a4b51d)

Reply:
> How do you display and use mnemonics in this menu or in the toolbar? Is there a trick to open this toolbar menu with a keyboard hotkey? To navigate in the toolbar?

No, I don't know how to navigate there using keys. `TabStop` seems to be `false`.

But I let Window always show mnemonics:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/6837ac9f-6487-4d79-bb42-d83674a4ce98)

Reply:
> I have the feeling I already asked it but don't remember the answer so asking again: How do you display and use mnemonics in this menu or in the toolbar? Is there a trick to open this toolbar menu with a keyboard hotkey? To navigate in the toolbar? TIWTL (Today I want to learn 😜 )
> 
> Because I'm only able to open with the mouse and it looks like: ![image](https://private-user-images.githubusercontent.com/460196/343854163-a902a186-0c37-4c94-aea4-d7921583044d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4MTcyMjksIm5iZiI6MTcxOTgxNjkyOSwicGF0aCI6Ii80NjAxOTYvMzQzODU0MTYzLWE5MDJhMTg2LTBjMzctNGM5NC1hZWE0LWQ3OTIxNTgzMDQ0ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwMVQwNjU1MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNDk5NTllY2JkODE4ZTg2YzIwZGIyNTZkNWVjNWJhNjg5N2ViZTllMzI3NTM3M2U3MmVhYzkxMDYyM2MzNWJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IV-vAm8amqWGnI6cHWMCj8LdlS2KNmnecSLyT1Syfa8)
> 
> @RussKie `o` mnemonic is already used by 'Open...' that's why @mstv was proposing `&n`

I'm going blind... Aging is not fun :(
Thank you.

To see underscores I'm tapping ALT twice (though evidentyly it's not helping).

Reply:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/7cb4ca01-3549-4a42-8b0f-1fed1b5f6c88)